psi4/src/psi4/dlpno/mp2.cc

```diff
@@ -1180,7 +1181,9 @@ double DLPNOMP2::compute_energy() {
 
     print_header();
 
+    timer_on("Setup");
```

Comment:
Could this be more descriptive? Setup does the PAOs?

Reply:
Yes, `setup()` does PAOs and also LMOs. I updated the function name to `setup_orbitals()`